.github/workflows/e2e_nightly.yml

```diff
@@ -0,0 +1,121 @@
+name: E2E Nightly
+
+# Remove before merge!
+on: [push, pull_request]
+  # Uncomment before merge!
+  # schedule:
```

Comment:
A scheduled build will work only on the default (currently: main) branch. I guess, you have to do a similar thing as Patryk did for long-fuzz tests (https://github.com/oneapi-src/unified-runtime/blob/main/.github/workflows/nightly.yml)

Reply:
Done